### ISSUE TITLE
ERROR:pastehunter.py:Unable to store xxxx

### ISSUE THREAD

**Issue by gitxmax** [kevthehermit/PasteHunter#20]
Tried rebuilding a few times, but, every time, this error comes up.
Up to this point, everything seems to work.
Would appreciate any help.

ERROR:pastehunter.py:Unable to store dxy4fUrc
 If of any help, here's what my Docker has running: https://i.imgur.com/6vlnFbP.png

**Comment by kevthehermit** [comment]
Is it failing to store every paste?

**Comment by gitxmax** [comment]
Yes, it does, tried different versions of Yara/Elastic/Docker, everytime it's the same

**Comment by acd62081** [comment]
I am not sure if you are still having this issue or not, but I was experiencing this same error with my Docker instance. I deleted and rebuilt the Docker images/containers and noticed that the docker-compose.yml had the network of 172.18.0.0/24 but the Dockerfile was waiting for 172.16.10.10:9200. Once I changed the IP address in the Dockerfile to 172.18.10.10:9200, everything started working fine. At that I point I downloaded the dashboard that I found here:  https://techanarchy.net/assets/images/2017/12/dashboard.zip
The dashboards and visualizations are working really well now. 

Thought I would share. Hopefully it helps.
